docs/adoption.md

```diff
@@ -87,13 +89,14 @@ Once you have added a remote backend to your Terraform and created a service pri
 
   1. Add the following secrets as your github repository secrets, these are only available to you if you have access to create a service principal so ensure to request these if the service principal is being created for you, (Note: navigate to your Service principal under Active Directory → App registrations → select your app registration and navigate to overview):
 
-      | Secret name              | Value                                                                                                         |
-      | ------------------------ | ------------------------------------------------------------------------------------------------------------- |
-      | `AZURE_AD_CLIENT_SECRET` | This is the client secret value that was generated for the service principal in section 4 of Create a service |
-      | `AZURE_AD_CLIENT_ID`     | This is the Application (client) ID                                                                           |
-      | `AZURE_AD_TENANT_ID`     | This is the Directory (tenant) ID                                                                             |
-      | `AZURE_SUBSCRIPTION_ID`  | Navigate to subscriptions and select the Subscription ID for your subscription                                |
-      | `STATIC_SITE_NAME`       | The name of your static site                                                                                  |
+      | Secret name              | Value                                                                                                                 | Mandatory? |
+      | ------------------------ | --------------------------------------------------------------------------------------------------------------------- | ---------- |
+      | `AZURE_AD_CLIENT_SECRET` | This is the client secret value that was generated for the service principal in section 4 of Create a service.        | YES        |
+      | `AZURE_AD_CLIENT_ID`     | This is the Application (client) ID                                                                                   | YES        |
+      | `AZURE_AD_TENANT_ID`     | This is the Directory (tenant) ID                                                                                     | YES        |
+      | `AZURE_SUBSCRIPTION_ID`  | Navigate to subscriptions and select the Subscription ID for your subscription                                        | YES        |
+      | `STATIC_SITE_NAME`       | The name of your static site                                                                                          | YES        |
+      | `CUSTOM_DOMAIN_NAME`     | The custom domain name you wish to attach to your static site. See [below](adoption.md#notes-adding-a-custom-domain). | NO         |
```

Comment:
This anchor ([below]) doesn't seem to work currently in the preview. Need something like this below: 
_`<a name="notes-adding-a-custom-domain"></a>`_

Reply:
Thanks for spotting. Updated in [c912de3](https://github.com/Newarkandsherwood/housing-repairs-online/pull/23/commits/c912de34ff4c5187330fa15c3bf689f74a35bbb3).